File: Majo/practicas/tabla_de_multiplicar.dart
import 'dart:io';

void main(){
  
  print('Ingresa el número del que deseas la tabla de multiplicar');
  int num = int.parse(stdin.readLineSync()!);
    print(' ');
    print('La tabla de multiplicar del número $num es:');
    print(' ');
    
  for (var i = 1; i <= 10; i++) {

    int ope = num*i;
    print('$num x $i = $ope');
  }
  

}

File: Majo/practicas/tabla_de_multiplicar.rb
puts "Ingresa el número del que deseas la tabla de multiplicar"
num = gets.chomp.to_i
puts " "
puts "La tabla de multiplicar del número #{num} es"
puts " "

for i in 1..10
    ope = num*i
    puts "#{num} x #{i} = #{ope}"
end    